packages/common-lib/src/components/FilterButton.js

```diff
@@ -35,11 +46,17 @@ export default function FilterButton({
             colorScheme='button'
             variant='outline'
             px='5'
+            _text={{
```

Comment:
Possible to use the same `H1`, `H2` that we created earlier here.

Reply:
that is in another PR